templates/edit_mod.html

```diff
@@ -173,17 +173,25 @@ <h2>Additional Authors</h2>
     </div>
     {% endif %}
 
-    <div class="container" style="margin-top: 2.5mm;">
-        <p>
-            <span class="col-md-10 form-group"><a target="_blank" href="/markdown">Markdown</a> is supported here.</span>
-            {% if not mod.ckan %}
-            <label>
-                <input id="ckan" name="ckan" style="margin-top: 8px;" type="checkbox"> Add mod to <a target="_blank" href="https://github.com/KSP-CKAN/CKAN">CKAN</a>
-                <span data-original-title="This makes it so that users can automate the installation of your mod. Your mod won't be added to CKAN until you publish it." class="glyphicon glyphicon-question-sign" data-toggle="tooltip" data-placement="left" title=""></span>
-            </label>
-            {% endif %}
-        </p>
-        <textarea class="form-control input-block-level" name="description" id="description" rows=25>{{ mod.description }}</textarea>
+    <div class="container">
+        <div class="row">
+            <div class="col-md-5"><a target="_blank" href="/markdown">Markdown</a> is supported here.</div>
+            <div class="col-md-5 form-group text-right">
+                <input id="ckan" name="ckan" type="checkbox"
+                    {% if mod.ckan %}
+                       checked="checked"
+                       {% if mod.published and not user.admin %}
+                           disabled="disabled"
+                       {% endif %}
+                    {% endif %}
+                />
+                <label for="ckan">Add mod to <a target="_blank" href="https://github.com/KSP-CKAN/CKAN">CKAN</a></label>
+                <span title="This triggers a pull request in the NetKAN repository to index your mod in the CKAN after you publish it." class="glyphicon glyphicon-question-sign" data-toggle="tooltip" data-placement="left"></span>
```

Comment:
Though if we do keep this tooltip, it might be easier for users to get up to speed on pull requests and repositories if there was a link to the recent ones:

https://github.com/KSP-CKAN/NetKAN/pulls?q=is%3Apr+author%3Anetkan-bot

Reply:
Yeah I wanted to add a link to NetKAN as well, but since it's a tooltip that only shows on mouse-over, you can't click it.
Maybe display this help text next to the checkbox all the time?

Reply:
Okay, I've changed it to a text below the checkbox that's hidden by default and can be toggled by a small button.
![image](https://user-images.githubusercontent.com/28812678/109215878-154b3100-77b4-11eb-9c01-7f310c100fdc.png)

The text is now longer to hopefully help unfamiliar modders to understand what's going on, and contains the above link.

Open to further suggestions.